core/sds-test-aspect-models/src/main/resources/valid/bamm_2_0_0/io.openmanufacturing.test.shared/1.0.0/AspectWithCollectionEntity.ttl

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2022 Robert Bosch Manufacturing Solutions GmbH
+    # Copyright (c) 2022 Robert Bosch Manufacturing Solutions GmbH
```

Comment:
Remove unnecessary indentation. Also see other model files below, this occurs multiple times.

Reply:
done